src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -432,14 +432,14 @@ export class ShelleyWallet implements WalletInterface {
     return changeAddress
   }
 
-  private async getAddressedChangeAddress(): Promise<{address: string; addressing: CardanoTypes.Addressing}> {
+  private getAddressedChangeAddress(): Promise<{address: string; addressing: CardanoTypes.Addressing}> {
     const changeAddr = this.getChangeAddress()
     const addressing = this.getAddressing(changeAddr)
 
-    return {
+    return Promise.resolve({
       address: changeAddr,
       addressing,
-    }
+    })
```

Comment:
This function probably does even need to be async / return Promise

Reply:
im not worried about this right now. im focusing on not changing behavior and we can fix this stuff later